src/routes/safe/store/actions/processTransaction.js

```diff
@@ -165,6 +167,11 @@ const processTransaction = ({
     closeSnackbar(pendingExecutionKey)
     showSnackbar(notificationsQueue.afterExecutionError, enqueueSnackbar, closeSnackbar)
 
+    const newPendingActions = Map([
```

Comment:
When I thought about doing this ticket I thought about simply mocking the confirmation (we have all the data for it) and adding `status: 'PENDING'`. Why did you choose this approach?

Reply:
So in general I thought about mocking the backend data to kind of "overtake" it. E.g add a mock cancellation transaction before the backend proceeds it. I think it may ease the data processing in the future

Reply:
I just didn't have that idea but how do you know if the user did and reject or accept action in order to update those labels? and what happens if for instance, you have two accounts, you confirm on the first one and then you move quickly to the second one and do another operation (accept/reject), the status of the first approval will be lost unless you save the userAddress in some way that's why I have a list of owners

Reply:
confirmation object has user address